eksrollup/lib/k8s.py

```diff
@@ -143,6 +143,35 @@ def cordon_node(node_name):
         logger.info("Exception when calling CoreV1Api->patch_node: {}".format(e))
 
 
+def taint_node(node_name):
+    """
+    Taint a kubernetes node to avoid new pods being scheduled on it
+    """
+
+    try:
+        config.load_incluster_config()
+    except config.ConfigException:
+        try:
+            config.load_kube_config()
+        except config.ConfigException:
+            raise Exception("Could not configure kubernetes python client")
+
+    configuration = client.Configuration()
+    # create an instance of the API class
+    k8s_api = client.CoreV1Api(client.ApiClient(configuration))
+    logger.info("Adding taint to k8s node {}...".format(node_name))
+    try:
+        taint = client.V1Taint(effect='NoSchedule', key= 'eks-rolling-update')
```

Comment:
The Travis CI Build Failed due to a styling issue on this line since the parameter 'key' has an extra space before 'eks-rolling-update'.

Remove the space and the space here and it should pass the Travis CI Test
`key= 'eks-rolling-update'`

Travis CI Error: _eksrollup/lib/k8s.py:164:57: E251 unexpected spaces around keyword / parameter equals_
Link to the Build Failure: https://travis-ci.org/github/hellofresh/eks-rolling-update/builds/694814928